browser/src/core/Socket.js

```diff
@@ -1533,6 +1534,7 @@ app.definitions.Socket = L.Class.extend({
 			}
 			if (this._map._docLayer._docType === 'presentation')
 				this._map._isCursorVisible = false;
+			this._map._docLayer._canonicalIdInitialized = false;
```

Comment:
I think it would be better to not modify "private" variable outside CanvasTileLayer.js.
Could you create method inside CanvasTileLayer.js which resets that value?
like: `resetCanonicalId()`